app/templates/components/new-course.hbs

```diff
@@ -23,6 +23,9 @@
     <label>{{t "general.academicYear"}}:</label>
 
     <select data-test-year onchange={{action "setYear" value="target.value"}}>
+      <option disabled hidden selected value="">
```

Comment:
My reading of the `hidden` attribute says this is invalid: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden thoughts?

It's also invalid to have more than one element selected this causes issues with keyboard navigation it should probably be `selected={{is-empty this.selectedYear}}` though I'm not sure about `is-empty`.

Reply:
Fixed.